### PR TITLE
options.format: lines (output packages line by line)

### PR DESCRIPTION
**Pull request by vanodevium**
#1282 

@raineorshine I have some problems with TS :(

Anyway, I'm trying. 

Is this correct implementation? Can I continue work with tests and readme?

### CODE REVIEW

File: src/lib/initOptions.ts [hunk 1]
```diff
@@ -64,12 +64,16 @@ async function initOptions(runOptions: RunOptions, { cli }: { cli?: boolean } =
...
-  const loglevel = options.silent ? 'silent' : options.verbose ? 'verbose' : options.loglevel
+  let loglevel = options.silent ? 'silent' : options.verbose ? 'verbose' : options.loglevel
 
   const json = Object.keys(options)
     .filter(option => option.startsWith('json'))
     .some(propertyOf(options))
 
+  if (options.format?.includes('shell') || options.format?.includes('lines') || options.format?.includes('comma')) {
+    loglevel = 'silent';
+  }
```
Comment (raineorshine) [inline src/lib/initOptions.ts:75]: You can do this logic in the ternary and avoid changing `const` to `let`.

**Review by raineorshine — CHANGES_REQUESTED** [review]

**Comment by raineorshine** [comment]
`--format unix` is ambiguous in its own way, and I don't want to imply that this feature is platform specific and can't be used by Windows users. `--format lines` describes the output: one update per line, without anything else. The help system can clarify further.

**Comment by vanodevium** [comment]
OK. I have done my best.

**Comment by vanodevium** [comment]
Thanx!

**Pull request merged**